Testscripts/Linux/LSVMBUS.sh

```diff
@@ -70,7 +70,7 @@ if [ "$os_GENERATION" -eq "1" ]; then
 fi
 
 # lsvmbus requires python
-which python || [ -f /usr/libexec/platform-python ] && ln -s /usr/libexec/platform-python /sbin/python
+which python || [ -f /usr/libexec/platform-python ] && ln -s /usr/libexec/platform-python /sbin/python || which python3 && ln -s $(which python3) /sbin/python
```

Comment:
Why are we creating softlink with python3 ? Is ```install_package``` python not working ?
Instead of creating a softlink for python, we can export the PATH so ```which``` command will pick it up.

Reply:
Thanks @johnsongeorge-w , I double check.

- python2 no exist any more on Ubuntu 20.04
- no need create softlink, it also works (I will update this in next PR), now it is harmless